src/DssSpell.sol

```diff
@@ -18,49 +18,244 @@ pragma solidity 0.6.12;
+            URN:  0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90,
+            gemID: "RWA005",
+            joinID: "MCD_JOIN_RWA005_A",
+            urnID: "RWA005_A_URN",
+            inputConduitID: "RWA005_A_INPUT_CONDUIT",
+            outputConduitID: "RWA005_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA005",
+            ilk: "RWA005-A",
+            ilk_string: "RWA005-A",
+            ilkRegistryName: "RWA005-A: Centrifuge: Fortunafi",
+            RATE: FOUR_PT_FIVE_PCT,
+            CEIL: 15 * MILLION,
+            PRICE: 17_199_394 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmbgDoPn6UcfSDENDqHLgatMFoqXikC8E8it9WaZXyLXmc"
+        });
+
+        // https://vote.makerdao.com/polling/QmdHV2tW#poll-detail
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x5E11E34b6745FeBa9449Ae53c185413d6EdC66BE,
+            GEM: 0x4EE03cfBF6E784c462839f5954d60f7C2B60b113,
+            OPERATOR: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            INPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            OUTPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            URN:  0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilk_string: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
+            RATE: TWO_PCT,
+            CEIL: 0 * MILLION,
+            PRICE: 0 * WAD,
+            MAT: 10_000,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues[4] memory collaterals = [RWA003, RWA004, RWA005, RWA006];
+
+        // Integrate RWA003-006
+        for (uint i = 0; i < collaterals.length; i++) {
+            integrateCentrifugeCollateral(collaterals[i]);
+        }
+
+        // Bump changelog version
+        DssExecLib.setChangelogVersion("1.9.2");
+    }
+
+    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
+        address MIP21_LIQUIDATION_ORACLE =
+            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
+
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
```

Comment:
Reminder that to amend the `doc` for `RWA006` we will need to call `init` again on the liquidation oracle. We should add a test to ensure that `doc` is not an empty string if the debt ceiling is greater than `0`.

Reply:
@offerijns you have time to make this test?  This is just so we don't get burned increasing a DC without replacing the IPFS hash of the SPV.

Reply:
@godsflaw @brianmcmichael I think the easiest is to replace this part of `_test_RWA_values()`:
```solidity
// Confirm pip value.
RwaLiquidationLike RwaLiqOracle = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
(,address pip_,,) = RwaLiqOracle.ilks(ilk);
DSValueAbstract pip = DSValueAbstract(pip_);
assertEq(uint256(pip.read()), price_);
```

With:
```solidity
// Confirm pip value.
RwaLiquidationLike RwaLiqOracle = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
(string memory doc,address pip_,,) = RwaLiqOracle.ilks(ilk);
assertEq(uint256(DSValueAbstract(pip_).read()), price_);

// Confirm doc value is set if price is larger than 0.
assertTrue(price_ == 0 || bytes(doc).length > 0);
```

I tested this locally and confirmed that it passes with the current spell, but fails if we set the price to > 0 without setting the `DOC` property.